examples/tfengine/generated/org_foundation/devops/variables.tf

```diff
@@ -11,12 +11,31 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 variable "admins_group" {
   type = object({
-    id = string
+    customer_id  = string
+    description  = string
+    display_name = string
+    exists       = string
+    id           = string
+    owners       = list(string)
+    managers     = list(string)
+    members      = list(string)
   })
-  description = "Group which will be given admin access to the folder or organization."
+  description = <<EOF
+    Group which will be given admin access to the folder or organization.
+
+    Fields:
+
+    * customer_id = Customer ID of the organization to create the group in.
+See <https://cloud.google.com/resource-manager/docs/organization-policy/restricting-domains#retrieving_customer_id>
+for how to obtain it.
+    * description = Description of the group.
+    * display_name = Display name of the group.
+    * exists = Whether or not the group exists already. It will be created if not.
+    * id = Email address of the group.
+    * owners = Owners of the group.
```

Comment:
what about managers and members?

Reply:
They're disabled at the schema, we agreed on the first PR to keep them in templates but leaving them commented on the schema.

I cannot access the description if it's disabled

Reply:
oh right. my bad.